src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  function getPartnerSex(personSex) {
+    return personSex === 'f' ? 'husband' : 'wife';
+  }
+
+  const marriedStatus = isMarried
+    ? `${partnerName} is my ${getPartnerSex(sex)}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
```

Comment:
Here bettween tags declarations you could've put some spaces so it is a bit easier to read. Hope it make sense)
<opening tag
  something 
 <closing tag

{ age something
 }
 
 {isMarried
 }